carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/webdriver/locator/internal/LocatingElementListHandler.java

```diff
@@ -65,10 +65,10 @@ public Object invoke(Object object, Method method, Object[] objects) throws Thro
 		// timeout before we find better solution
 		// Pros: super fast regression issue which block UI execution
 		// Cons: there is no way to manage timeouts in this places
-    	if (!waitUntil(ExpectedConditions.or(ExpectedConditions.presenceOfElementLocated(by),
-    			ExpectedConditions.visibilityOfElementLocated(by)))) {
-    		LOGGER.error("List is not present: " + by);
-    	}
+//    	if (!waitUntil(ExpectedConditions.or(ExpectedConditions.presenceOfElementLocated(by),
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.53.207.202/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)